packages/types/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Nagging, but if we're calling the package particle-types, please name the directory to match the package.